ArchiSteamFarm/Plugins/PluginsCore.cs

```diff
@@ -81,7 +84,7 @@ public static class PluginsCore {
 
 		string resourceName = OS.GetOsResourceName(objectName);
 
-		if (ActivePlugins.Count == 0) {
```

Comment:
Please revert all those changes, while the additional property is useful for outside callers, I always prefer lowest-level implementation for inside class, since it's easier to refactor it in the future.